### ISSUE TITLE
unable to run ktail

### ISSUE THREAD

**Issue by lakamsani** [atombender/ktail#21]
~ ktail --version
ktail 1.0.1

```
 ktail  foo-pod -n foo
stat /Users/vamsee.lakamsani/.kube/lab.yaml:/Users/vamsee.lakamsani/.kube/config no such file or directory%
```

`kubectl` works fine in the same terminal.  Context is already set with cluster and namespace using `kubectl config use-context`. Same issue with the `--context` flag as well. Somethi

**Comment by atombender** [comment]
That's an old version. Try installing the latest one.

**Comment by lakamsani** [comment]
That run was from the binary installed using `hombrew` instructions. 

Jut pulled the latest from the main branch and built it. Shows the same version. 

```
➜  ktail git:(main) git remote -v
origin	https://github.com/atombender/ktail (fetch)
origin	https://github.com/atombender/ktail (push)
➜  ktail git:(main) go build
➜  ktail git:(main) ./ktail --version
ktail 1.0.1
```

Should I build from a different branch?

**Comment by lakamsani** [comment]
Another engineer on my team is seeing the same problem.

**Comment by lakamsani** [comment]
I clone this repo for the first time today. So `git pull` is not the issue. https://github.com/atombender/ktail/blob/v1.3.0/version.go#L7 still shows 1.0.1 although the releases directory shows 1.3.0. Anyways, neither local build nor binary download work. We will look into using `stern` or `kubetail` unless you have any other suggestions to try. Thanks.

**Comment by lakamsani** [comment]
We are using `go version go1.19.1 darwin/amd64` btw.

**Comment by atombender** [comment]
I see your problem now; you have two files in your `KUBECONFIG` path. I've just pushed a fix tagged as 1.3.1 that should fix this. Also, the version printed on `--version` was out of date.

**Comment by lakamsani** [comment]
That works. Thank you very much.
